staging/src/k8s.io/client-go/examples/in-cluster/main.go

```diff
@@ -43,6 +44,21 @@ func main() {
 			panic(err.Error())
 		}
 		fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+
+		// Examples for error handling:
```

Comment:
A pity we cannot import a main package. I would rename `examples/out-of-cluster` to `examples\crud`, move the meat of main.go to `examples/crud/pkg` and import that from a minimal `examples/in-cluster/main.go`. Does that make sense? As a follow-up is fine.